src/main/java/ShortSet.java

```diff
@@ -1,9 +1,10 @@
+import java.util.*;
+
 public class ShortSet {
```

Comment:
You fixed the following error in this Pull Request:

RecommenderTest/src/main/java/ShortSet.java:3: error: cannot find symbol
    Set<Short> s = new HashSet<>();
    ^
  symbol:   class Set
  location: class ShortSet

Google's Error Prone static analysis tool can be used to find more issues similar to this. Check out http://errorprone.info for more information.